ynput/ayon-deadline
# Support `LimitGroups` and `MachineLimit` Houdini Deadline

## Changelog Description
Support `LimitGroups` and `MachineLimit` Houdini Deadline

## Snippets
 | Houdini Deadline Settings | publisher UI |
|--|--|
| ![image](https://github.com/user-attachments/assets/6187347f-4d49-46a8-aa37-3b6a2f8f577f) | ![image](https://github.com/user-attachments/assets/e4f7c17d-bad1-4408-9207-b99b34874d84) |

Submitted Job info
| `MachineLimit` | `LimitGroups`  |
|--|--|
| ![image](https://github.com/user-attachments/assets/d6212708-a3d0-4cf2-95fb-47819e7f8403) |  ![image](https://github.com/user-attachments/assets/f59db922-279f-4ee2-97b8-c42b5fe365bd) |


## Testing notes:

1. Launch Houdini Lacuncher
2. Publish Render and Cache to farm via publisher while setting limit groups and machine limit.
3. ensure the settings are propagated to the deadline job
